### ISSUE TITLE
Blank pages only after deploying to github pages

### ISSUE THREAD

**Issue by Gabirel** [theme-next/hexo-theme-next#1200]
## Issue Checklist 

- [x] I have executed `hexo clean`, cleared the browser cache and disabled all CDN services (e.g. Cloudflare Rocket Loader), this issue still exists.
- [x] I have confirmed that this issue only appears in theme NexT, but cannot be reproduced in other themes (e.g. default theme landscape). In other words, prove that this is a NexT issue, not a issue from Hexo.
- [x] I have already viewed [NexT Documentation Site](http://theme-next.org/docs/) and [Hexo Docs page](https://hexo.io/docs/), especially [Troubleshooting](https://hexo.io/docs/troubleshooting.html) part.
- [x] I have already viewed the latest Roadmap on GitHub and searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue), which does not help me.

***

## Expected behavior 
Act normally like in localhost

## Actual behavior 

blank pages
- Links to demo site with this issue: https://godeep.pro/
- Links to repository or source code of the blog: https://github.com/Gabirel/Gabirel.github.io

I am not the first time to use next theme. But I get this issue and I can't solve it. Don't know why since this never happens before when I am using before 5.x version of next

## Steps to reproduce the behavior 
1. Init blog
2. Use next theme
3. Deploy to github pages

## Environment Information
OS: MacOS X
### Node.js and NPM Information

```
v11.14.0
6.10.3
```

### Package dependencies Information

```json
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.9.0"
  },
  "dependencies": {
    "hexo": "^3.9.0",
    "hexo-deployer-git": "^2.0.0",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index": "^0.2.1",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-marked": "^2.0.0",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.3"
  }
}
```

### Hexo Version

```
hexo: 3.9.0
hexo-cli: 3.0.0
os: Darwin 18.7.0 darwin x64
node: 11.14.0
v8: 7.0.276.38-node.18
uv: 1.27.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 67
nghttp2: 1.37.0
napi: 4
llhttp: 1.1.1
http_parser: 2.8.0
openssl: 1.1.1b
cldr: 34.0
icu: 63.1
tz: 2018e
unicode: 11.0
```

### Hexo Configuration

```yml
# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: Hexo
subtitle:
description:
keywords:
author: John Doe
language: en
timezone:

# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: http://godeep.pro
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link: true # Open external links in new tab
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace:

# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date

# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: hexo-theme-next

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
  type: git
  repo: git@github-hexo.com:Gabirel/Gabirel.github.io.git
  branch: master
```

### NexT Configuration

```yml
Default Value!! Haven't changed anything yet since this is a startup.
```

## Other Information 
Github pages:
![image](https://user-images.githubusercontent.com/12933851/66177852-c324df80-e695-11e9-8a15-77495dc701f7.png)

localhost:
![image](https://user-images.githubusercontent.com/12933851/66177890-dcc62700-e695-11e9-87a5-1f820bc25fd5.png)

**Comment by Gabirel** [comment]
1. No error found in website after deployed
2. Do not exclude lib folder from global git ignore files(https://github.com/theme-next/hexo-theme-next/issues/37)

`.gitignore_global`
```git
####################################
######## OS generated files ########
####################################
.DS_Store
.DS_Store?
*.swp
._*
.Spotlight-V100
.Trashes
Icon?
ehthumbs.db
Thumbs.db
```

**Comment by stevenjoezhang** [comment]
It seems that you have enabled Rocket Loader.

![屏幕快照 2019-10-04 上午11 50 38](https://user-images.githubusercontent.com/16272760/66179839-0d5d8f00-e69d-11e9-8b86-7cde9d1b3110.png)

Could you please answer this question: why do you check this checkbox?
> - [x] I have executed hexo clean, cleared the browser cache and disabled all CDN services (e.g. Cloudflare Rocket Loader), this issue still exists.

**Comment by stevenjoezhang** [comment]
Duplicate of #1147, #1170

**Comment by Gabirel** [comment]
Problem solved. Thanks.

**Comment by stevenjoezhang** [comment]
Fixed in https://github.com/theme-next/hexo-theme-next/pull/1470
